core/runtime/jsruntime.rs

```diff
@@ -1180,6 +1180,18 @@ impl JsRuntime {
     state.op_state.clone()
   }
 
+  /// Returns the runtime's op names, ordered by OpId.
+  pub fn op_names(&self) -> Vec<String> {
```

Comment:
Since `OpDecl.name` is a `&'static str` maybe we can return `Vec<&'static str>` to avoid excessive copies?

Reply:
makes sense. done.